index.js

```diff
@@ -23,19 +23,17 @@ class Plugin extends PluginBtp {
     this._account = opts.account
     this._db = opts.db || 'machinomy_db'
     this._provider = opts.provider || 'http://localhost:8545'
-    this._minimumChannelAmount = opts.minimumChannelAmount || 100
+    this._minimumChannelAmount = opts.minimumChannelAmount || new BigNumber(100)
```

Comment:
Should `this._minimumChannelAmount` be typeof string or BigNumber?

Reply:
It should be of type BigNumber, because it will be [used directly](https://github.com/michielbdejong/ilp-plugin-ethereum-asym-client/blob/519721c555fb63fa9cb5336cf3cf22d5c4def21e/index.js#L54) to [open a Machinomy channel](https://github.com/machinomy/machinomy/blob/6be1cff10b32cd76b5a2911ed8ec9e168362c251/lib/channel_manager.ts#L33)